Sources/SwiftParser/TokenPrecedence.swift

```diff
@@ -22,7 +22,7 @@ enum TokenPrecedence: Comparable {
   case identifierLike
   /// Keywords and operators that can occur in the middle of an expression
   case exprKeyword
-  /// A token that starts a bracketet expression which typically occurs inside
+  /// A token that starts a bracket expression which typically occurs inside
```

Comment:
This is meant to be `bracketed`. Same in a couple more cases in this file.

```suggestion
  /// A token that starts a bracketed expression which typically occurs inside
```